active_directory/assets/configuration/spec.yaml

```
@@ -0,0 +1,69 @@
+name: Active Directory
+files:
+- name: active_directory.yaml
+  options:
+  - template: instances
+    options:
+    - name: host
+      required: true
+      description: |
+        Host that the Datadog active directory check connects to.
+        "." means the current host
+      value:
+        example: .
+        type: string
+    - name: username
+      required: false
+      description: Username from the credentials needed to connect to the host.
+      value:
+        type: string
+    - name: password
+      required: false
+      description: Password from the credentials needed to connect to the host.
+      value:
+        type: string
+    - name: additional_metrics
+      required: false
+      description: |
+        The additional metrics is a list of items that represent additional counters to collect.
+        Each item is a list of strings, formatted as follows:
+
+        ['<COUNTERSET_NAME>', <COUNTER_INSTANCE_NAME>, '<COUNTER_NAME>', <METRIC_NAME>, <METRIC_TYPE>]
+
+        <COUNTERSET_NAME> is the name of the PDH Counter Set (the name of the counter)
+        <COUNTER_INSTANCE_NAME> is the specific counter instance to collect, for example
+                                "Default Web Site".  Specify 'none' For all instances of
+                                the counter.
+        <COUNTER_NAME> is the individual counter to report
+        <METRIC_NAME> is the name you want to show up in Datadog
+        <METRIC_TYPE> is from the standard choices for all agent checks, such as gauge,
+                      rate, histogram or counter
```

```suggestion
                      rate, histogram, or count.
```
Counter is deprecated